Client/CMakeLists.txt

```diff
@@ -1,17 +1,22 @@
-cmake_minimum_required(VERSION 3.19)
+cmake_minimum_required(VERSION 3.25)
 
 find_package(Thallium REQUIRED)
 find_package(spdlog REQUIRED)
 
+#message("ChronologClient CMakeLists - CMAKE_SOURCE_DIR=${CMAKE_SOURCE_DIR}")
+#message("ChronologClient CMakeLists - CMAKE_CURRENT_SOURCE_DIR=${CMAKE_CURRENT_SOURCE_DIR}")
+message("Building CMAKE_CURRENT_SOURCE_DIR=${CMAKE_CURRENT_SOURCE_DIR}")
+message("build target : chronolog_client")
+
 # Define the header files to be copied
 set(HEADER_FILES
```

Comment:
please 
1) rename HEADER_FILES into PUBLIC_HEADER_FILES and move this definition to the installation section 
( line 52)

2) remove lines 19 - 28 
3) use CMAKE_INSTALL_INCLUDE_DIR & CMAKE_INSTAL_LIB_DIRECTORY  as install DESTINATIONS